### DISCUSSION TITLE
以下不是github的官网网站，为什么不会被封

### DISCUSSION THREAD

**Discussion by yuan90gege** [community/community#53392] [Accessibility]
### Select Topic Area

Product Feedback

### Body

【GitHub学生包自助认证2.0】正式上线

自助认证地址：https://githubedu.com/
授权码购买地址：https://shop.githubedu.com/
自助认证教程地址：https://kdocs.cn/l/ca78l3iuugxA
权益使用B站教程地址：https://space.bilibili.com/395446863/video
权益使用博客教程地址：https://xiaoma.cool/2022/10/26/Pro%E7%BB%91%E5%AE%9AJetbrains%E3%80%81Copilot%E3%80%81Termius/
这个不是GitHub的官网教育网站，为什么存在这么久了不会被封掉呢？

**Comment by ghostinhershell** [comment]
Hi @yuan90gege, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
